scripts/setup/create_multinode.sh

```diff
@@ -100,7 +101,7 @@ function setup_master() {
 
     clone_loader $MASTER_NODE
 
-    MN_CLUSTER="./vhive/scripts/cluster/create_multinode_cluster.sh ${OPERATION_MODE}"
+    MN_CLUSTER="pushd ~/vhive/scripts > /dev/null && ~/vhive/scripts/setup_tool create_multinode_cluster ${OPERATION_MODE} && popd > /dev/null"
```

Comment:
You cd'd to vhive/scripts jsut to use absolute path. You can just run ./setup_tool instead.